.github/workflows/build-test.yml

```diff
@@ -0,0 +1,26 @@
+name: Java CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'adopt'
```

Comment:
adopt JDK stop releasing new Docker images about 9 months ago https://hub.docker.com/_/eclipse-temurin seems the better replacement

Reply:
I used the one defined in https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-maven#using-the-maven-starter-workflow

We might need to contribute also in the github docs :)

I'll update it with the proposal, although I cannot find the way to test the GitHub actions for the very first time until they are not merged into the target branch